mobx_codegen/lib/src/template/computed.dart

```diff
@@ -10,7 +10,7 @@ class ComputedTemplate {
   @override
   // ignore: prefer_single_quotes
   String toString() => """
-  Computed<$type> $computedName;
+  Computed<$type>? $computedName;
```

Comment:
Why are you forcing nullable here for the property ? The Computed is always non-nullable, instead the value it produces can potentially be nullable depending on how the user sets it up. Please remove the `?' here

Reply:
But we need to support `$computedName ??= Computed()`, so it should be nullable

Reply:
That is for the name of the computed property, not the Computed type

Reply:
I am confused... if remove "?" then error.

Reply:
Normally (notice the type is `int?`here, but often it is things like `int` in other places:

![image](https://user-images.githubusercontent.com/5236035/110243585-fc056a00-7f95-11eb-80b2-1cf566806002.png)


If no "?"

![image](https://user-images.githubusercontent.com/5236035/110243568-f019a800-7f95-11eb-8593-22357bf4c4ed.png)

If use late:

![image](https://user-images.githubusercontent.com/5236035/110243562-e42de600-7f95-11eb-8c93-5354985fdc1c.png)

Reply:
ya I see the issue now. Computed is created lazily on the first access, so I think late is the right approach. Not sure why greyed out text. Can you replace with `late` for now?

Reply:
the initialization of the computed variable can be move to the declaration once we make it lazy. In fact we can make it `final lazy`